discord/message.py

```diff
@@ -369,6 +390,45 @@ async def read(self, *, use_cached: bool = False) -> bytes:
         data = await self._http.get_from_cdn(url)
         return data
 
+    async def read_chunked(
+        self, *, use_cached: bool = False, chunksize: int | None = None
+    ) -> AsyncGenerator[bytes]:
+        """|coro|
+
+        Retrieves the content of this attachment in chunks as a :class:`AsyncGenerator` object of bytes.
+
+        Parameters
+        ----------
+        use_cached: :class:`bool`
+            Whether to use :attr:`proxy_url` rather than :attr:`url` when downloading
+            the attachment. This will allow attachments to be saved after deletion
+            more often, compared to the regular URL which is generally deleted right
+            after the message is deleted. Note that this can still fail to download
+            deleted attachments if too much time has passed, and it does not work
+            on some types of attachments.
+        chunksize: :class:`int`
+            The maximum size of each chunk to process.
+
+        Returns
+        -------
+        :class:`AsyncGenerator`
+            Generator of the contents of the attachment.
```

Comment:
I think we use `Yields`: https://docs.pycord.dev/en/v2.6.1/api/abcs.html#:~:text=otherwise%20False.-,YIELDS,-%3A

```suggestion
        Yields
        -------
        :class:`bytes`
            A chunk of the file.
```